specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/stable/2022-03-01/FabricLocation.json

```diff
@@ -0,0 +1,236 @@
+        "parameters": [
+          {
+            "$ref": "../../preview/2016-05-01/Fabric.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../preview/2016-05-01/Fabric.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/FabricLocationParameter"
+          },
+          {
+            "$ref": "../../preview/2016-05-01/Fabric.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/FabricLocation"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "../../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Fabric.Admin/fabricLocations": {
+      "get": {
+        "x-ms-examples": {
+          "Returns a list of all fabric locations.": {
+            "$ref": "./examples/FabricLocation/List.json"
+          }
+        },
+        "tags": [
+          "FabricLocations"
+        ],
+        "description": "Returns a list of all fabric locations.",
+        "operationId": "FabricLocations_List",
+        "parameters": [
+          {
+            "$ref": "../../preview/2016-05-01/Fabric.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../preview/2016-05-01/Fabric.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "../../preview/2016-05-01/Fabric.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../preview/2016-05-01/Fabric.json#/parameters/FilterParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/FabricLocationList"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "../../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "#/definitions/FabricLocation"
+      }
+    }
+  },
+  "definitions": {
+    "FabricLocation": {
+      "description": "Fabric Location resource.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "description": "Properties of a FabricLocation.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/FabricLocationModel"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../preview/2016-05-01/Fabric.json#/definitions/Resource"
```

Comment:
>../../preview/2016-05-01 [](http://example.com/codeflow?start=19&length=24)

This doesn't block ARM signoff, but it's generally a better practice to not have API v2 reference files from API v1. You could just copy the Fabric.json file forward.